src/main/java/mate/academy/resources/init_db.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE books (
+    id BIGINT NOT NULL AUTO_INCREMENT,
+    title VARCHAR(255),
+    price BIGINT,
```

Comment:
BIGINT contains numbers in integer type. Use type that contains numbers with floating point.